JumpTheQueue/index.asciidoc

```diff
@@ -1,4 +1,12 @@
 = Jump start an end-to-end devonfw sample application.
+
+[tags]
+--
+technology=java;devon4j;devon4ng
```

Comment:
I would prefer to use proper capitalization rather than just lowercase everything. 
For devon4j and devon4ng this is fine, but for Java, for example, I would use the first letter capitalized since it is a proper name. Please also check the other tutorials.